contracts/BatchRequestsForFinalizeData.sol

```diff
@@ -2,32 +2,29 @@
 pragma solidity ^0.8.19;
 
 import {IOracle} from '@defi-wonderland/prophet-core-abi/solidity/interfaces/IOracle.sol';
-import {IModule} from '@defi-wonderland/prophet-core-abi/solidity/interfaces/IModule.sol';
 
 /**
  * @title BatchRequestsForFinalizeData contract
- * @notice This contract is used to get batch requests data from the oracle contract
+ * @notice This contract is used to get batch requests min data from the oracle contract to finalize the requests
```

Comment:
What's min data?

Reply:
I referred to the minimum amount of data needed to finish a request.